drivers/lcd/ssd1680.c

```diff
@@ -722,10 +715,11 @@ static int ssd1680_configuredisplay(struct ssd1680_dev_s *priv)
 }
 
 /****************************************************************************
- * Name:  ssd1680_update_all
+ * Name:  ssd1680_update_all_and_redraw
```

Comment:
why not rename ssd1680_update_all_and_redraw to ssd1680_redraw?

Reply:
Updating drivers memory is postponed till redrawing. Previous name was confusing for me. <aybe it is better to call this function ssd1680_update_memory_and_redraw ?

Reply:
Ok, it's fine to keep as the current change. Please address other comments, thanks.